specs/kafka/SOURCES/checksum.sha512

```diff
@@ -0,0 +1,6 @@
+ea0522ed4bcc7648c6a22b0fb14d54ef63acf9c0bbcdbb6d8e26904270e4718cba0a7ceabd1b325183157cc978bea9e40d76824ae034ca00b82178ad1cd1b128  2.7.0.tar.gz
```

Comment:
@gongled `checksum.sha512` is not required for the build process. This file automatically generated by `rpmbuilder` while building packages. Also, this spec doesn't have a macro for checking these checksums.